app/views/devise/registrations/new.html.erb

```diff
@@ -14,11 +14,8 @@
   </div>
 
   <div class="field">
-    <%= f.fields_for :department do |d| %>
-      <%= d.label :department %>
-      <%= d.select :name,
-          Department::DEPARTMENT, class: "department dropdown"%>
-    <% end %>
+      <%= f.label :department_id %>
+      <%= collection_select(:department_id, :department_id, Department.all, :id, :name) %>
```

Comment:
That's right, there is a field on the user table for department_id. I know we just want to send the integer of the department_id. I don't think using collection_select is what we're looking for in this case, based off what you're saying? What should we use instead to display the select items and only send the id rather than the full hash or send an object?